src/NzbDrone.Core/MediaFiles/EpisodeImport/Aggregation/Aggregators/AggregateEpisodes.cs

```diff
@@ -10,6 +10,8 @@ namespace NzbDrone.Core.MediaFiles.EpisodeImport.Aggregation.Aggregators
 {
     public class AggregateEpisodes : IAggregateLocalEpisode
     {
+        public int Order => 1;
```

Comment:
Since most of these don't have a strict order I think it'd make sense to group everything that doesn't have a requirement at `1` and then things that do at `2`. That way multiple files don't need to be changed to insert something later.